src/Symfony/Component/Filesystem/Tests/FilesystemTestCase.php

```diff
@@ -27,17 +27,43 @@ class FilesystemTestCase extends \PHPUnit_Framework_TestCase
      */
     protected $workspace = null;
 
-    private static $symlinkOnWindows = null;
+    /**
+     * @var null Flag for hard links on Windows
+     */
+    protected static $linkOnWindows = null;
```

Comment:
should be private, and imo there's no need for the docblock here (same below for `$symlinkOnWindows`)